src/app/lib/README.md

```diff
@@ -41,3 +41,94 @@ Here are the font sizes in `px`s:
 | Minion       | 12            | 16              | 12            | 16              | 12            | 16              | Article category                                                 |
+
+# Layout requirements
+
+As per the [GEL grid Guidelines](https://www.bbc.co.uk/gel/guidelines/grid) we have the following columns:
+
+- 240px - 399px (6 column) 
+- 400px - 599px (6 column)
+- 600px - 1007px (6 column)
+- 1008px - 1279px (8 column)
+- 1280px+ (10 column)
+
+The following is a breakdown of the article layout design for given breakpoints:
+
+**Between 1px - 599px**
+
+- left and right margin of 8px
+- article content full width of the page EG: all columns used
+- *See Figures 1 and 2*
+
+**Between 600px - 1007px**
+
+- left and right margin of 16px
+- of 6 columns the article body content spans the central 4
+- *See Figure 3*
+
+**Between 1008px - 1279px**
+
+- left and right margin of 16px
+- the 8 columns are restricted to a width of 1008px
+  EG: at a 1208px viewport the layout should be { 100px | 8 columns | 100px } where the 100px are flexible making up the difference between the 1008px columns max width and the viewport
+- of 8 columns the article body content spans the central 6
+- *See Figure 4*
+
+**Between 600px - 1007px**
+
+- left and right margin of 16px
+- the 10 columns are restricted to a width of 1280px
+  EG: at a 1400px viewport the layout should be { 60px | 10 columns | 60px } where the 60px are flexible making up the difference between the 1280px columns max width and the viewport
+- of 10 columns the article body content spans the central 6
+- *See Figure 5*
+
+## Product requirements
+
+We have requirements for components within the article body to go "full bleed" in which they will break out of all defined columns and fill the entire viewport. This is simple to achieve for viewports under 1008px as the columns span the entire width of the viewport, however for viewports greater than 1007px this is more complex as we need to limit the max width of the columns whilst still supporting the ability for indivdual components to expand outside of the max width. For details see "article layout design for given breakpoints" in Layout Requirements.
+
+We also want flexiblity when placing components such as a single onward journey, and potentially adverts or similar, alongside the article body content. In the future the placement of this type of component will be content aware and potentially personalised. This requires another level of flexibilty.
+
+See Figure 7 for an example of the potential layout we may have to support. 
+
+## Problems
+
+As we want content to go "full bleed" we cannot use max-width to define the 1008|1280px column layouts. Also as full bleed spans to the very edge of the viewport each indivdual component will need to know about the layout margins.
```

Comment:
`indivdual` => `individual`

And - _would_ we want individual components to know about the layout margins? Feels like components should be dumb, ie "you have this much space to fill". I think that's probably what you mean already, just reads ambiguously to me!